packages/datadog-instrumentations/src/cucumber.js

```diff
@@ -432,22 +404,106 @@ function getWrappedRunTest (runTestFunction) {
 
         testSuiteCodeCoverageCh.publish({
           coverageFiles,
-          suiteFile: testFileAbsolutePath
+          suiteFile: testFileAbsolutePath,
+          testSuitePath
         })
         // We need to reset coverage to get a code coverage per suite
         // Before that, we preserve the original coverage
         mergeCoverage(global.__coverage__, originalCoverageMap)
         resetCoverage(global.__coverage__)
       }
 
-      testSuiteFinishCh.publish(testSuiteStatus)
+      testSuiteFinishCh.publish({ status: testSuiteStatus, testSuitePath })
     }
 
     return runTestCaseResult
   }
 }
 
-// From 7.3.0 onwards, runPickle becomes runTestCase
+function getWrappedParseWorkerMessage (parseWorkerMessageFunction) {
```

Comment:
ℹ️ `getWrappedParseWorkerMessage` is for intercepting messages from the workers and: 
* flush test data coming from the workers (workers now use the interprocess exporter)
* start / finish test suites